helpers/UtilsLogging.h

```diff
@@ -2,6 +2,13 @@
 
 #include <syscall.h>
 
+#undef LOGINFO
+#undef LOGWARN
+#undef LOGERR
+#undef LOG_DEVICE_EXCEPTION0
+#undef LOG_DEVICE_EXCEPTION1
+#undef LOG_DEVICE_EXCEPTION2
+
```

Comment:
please remove

Reply:
This can be removed later, after #2998 is merged. These macro are defined in both utils.h and UtilsLogging.h
SystemServices.h does include both (the latter through UtilsThreadRAII.h)